SU2_CFD/include/solvers/CScalarSolver.inl

```diff
@@ -404,6 +391,38 @@ void CScalarSolver<VariableType>::ImplicitEuler_Iteration(CGeometry* geometry, C
   CompleteImplicitIteration(geometry, solver_container, config);
 }
 
+template <class VariableType>
+void CScalarSolver<VariableType>::ExplicitEuler_Iteration(CGeometry* geometry, CSolver** solver_container,
+                                                          CConfig* config) {
+
+  const auto flowNodes = solver_container[FLOW_SOL]->GetNodes();
+
+  /*--- Local residual variables for current thread ---*/
+  su2double resMax[MAXNVAR] = {0.0}, resRMS[MAXNVAR] = {0.0};
+  unsigned long idxMax[MAXNVAR] = {0};
```

Comment:
@pcarruscag isnt this the #1445 problem as the function has a template parameter? I.e. the `= {0.0}` init should be removed

Reply:
Similar things existed and compiled, I think the problem is member variables of class templates, these are local variables of a function.